packages/byods/src/http-utils/index.ts

```diff
@@ -34,6 +35,11 @@ async function request<T>(url: string, options: HttpRequestInit = {}): Promise<A
     throw new Error(`HTTP Error Response: ${response.status} ${response.statusText}`);
   }
 
+  // Handle 204 No Content responses
+  if (response.status === 204) {
```

Comment:
This is important, for DELETE operations, for some reason it gave `204` code in the response. As a result there was no body and hence when we tried `response.json` in line 43 via fetch, it gave syntax error, hence I added this workaround.